lib/src/parse/css.dart

```diff
@@ -86,16 +87,38 @@ class CssParser extends ScssParser {
   ImportRule _cssImportRule(LineScannerState start) {
     var urlStart = scanner.state;
     var url = switch (scanner.peekChar()) {
-      $u || $U => dynamicUrl() as StringExpression,
+      $u || $U => switch (dynamicUrl()) {
+          StringExpression string => string.text,
+          InterpolatedFunctionExpression(
```

Comment:
I think this would work for most of the cases. However, this would attempt to parse a "interpolated" string. In pure css, technically I can do `@import url("a.css#{url-with-a-weird-hash}");` and it should still be valid, but I suppose this will throw interpolation is not allowed in css?

Reply:
In other words, strings in plain css should probably never be parsed as interpolated, even if they contain sass interpolation syntax.

Reply:
This code isn't actually parsing anything, it's just handling the various formats that the `dynamicUrl()` function can return. But yes, interpolations will throw errors in plain CSS mode even when they are technically valid CSS—largely because LibSass incorrectly interpreted `.css` files as SCSS, so we want to highlight places where that's not working. It might be worth revisiting that decision for Dart Sass 3.0.0, but probably not for 2.0.0.